invenio_accounts/profiles/schemas.py

```diff
@@ -24,10 +24,12 @@ def validate_visibility(value):
 
 def validate_locale(value):
     """Check if the value is a valid locale."""
-    locales = current_app.extensions["invenio-i18n"].get_locales()
-    locales = [locale.language for locale in locales]
+    locales = current_app.extensions.get("invenio-i18n")
```

Comment:
Why this change? `i18n` should be one base module always available.

Reply:
either invenio-communities or invenio-access has tests which make a call to this method and crash as there is apparently no instance of `invenio-i18n`

Reply:
Then, IMO, we should add `i18n` as dependency on the tests